sentences/fr/media_player_HassSetVolume.yaml

```diff
@@ -0,0 +1,8 @@
+language: fr
+intents:
+  HassSetVolume:
+    data:
+      - sentences:
+          - "<regle>[ le volume de ][<le>]{name} à {volume:volume_level}<pourcent>"
```

Comment:
Why do you have spaces inside the optional chuck of text ? 
Shouldn't it be 
`<regle> [le volume de] [<le>]{name} à ...` ?

Reply:
Well, no valuable reason for that, I can change it